features/admin.actions.v1/pages/actions.tsx

```diff
@@ -0,0 +1,363 @@
+} from "@wso2is/core/models";
+import { addAlert } from "@wso2is/core/store";
+import { DocumentationLink, GenericIcon, PageLayout, ResourceGrid, useDocumentation } from "@wso2is/react-components";
+import React, {
+    Fragment,
+    FunctionComponent,
+    ReactElement,
+    ReactNode,
+    useEffect,
+    useMemo
+} from "react";
+import { useTranslation } from "react-i18next";
+import { useDispatch, useSelector } from "react-redux";
+import { Dispatch } from "redux";
+import useGetActionTypes from "../api/use-get-action-types";
+import { ActionsConstants } from "../constants/actions-constants";
+import { ActionType, ActionTypeCardInterface, ActionTypesCountInterface } from "../models/actions";
+import "./actions.scss";
+
+/**
+ * Props for the Server Configurations page.
+ */
+type ActionTypesListingPageInterface = IdentifiableComponentInterface;
+
+/**
+ * Action Types listing page.
+ *
+ * @param props - Props injected to the component.
+ * @returns Action Types listing page component.
+ */
+export const ActionTypesListingPage: FunctionComponent<ActionTypesListingPageInterface> = ({
+    "data-componentid": _componentId = "action-types-listing-page"
+}: ActionTypesListingPageInterface): ReactElement => {
+
+    const actionsFeatureConfig: FeatureAccessConfigInterface = useSelector((state: AppState) =>
+        state.config.ui.features?.actions);
+
+    const { t } = useTranslation();
+    const { getLink } = useDocumentation();
+    const dispatch: Dispatch = useDispatch();
+
+    const {
+        data: actionTypesConfigs,
+        isLoading: isActionTypesConfigsLoading,
+        error: actionTypesConfigsRetrievalError
+    } = useGetActionTypes();
+
+    const typeCounts: ActionTypesCountInterface = useMemo(() => {
+        const actionTypeCounts: ActionTypesCountInterface = {};
+
+        if (actionTypesConfigs) {
+            for (const actionType of actionTypesConfigs) {
+                switch (actionType.type) {
+                    case ActionType.PRE_ISSUE_ACCESS_TOKEN:
+                        actionTypeCounts.preIssueAccessToken = actionType.count;
+
+                        break;
+                    case ActionType.PRE_UPDATE_PASSWORD:
+                        actionTypeCounts.preUpdatePassword = actionType.count;
+
+                        break;
+                    case ActionType.PRE_UPDATE_PROFILE:
+                        actionTypeCounts.preUpdateProfile = actionType.count;
+
+                        break;
+                    case ActionType.PRE_REGISTRATION:
+                        actionTypeCounts.preRegistration = actionType.count;
+
+                        break;
+                    default:
+                        break;
+                }
+            }
+
+            return actionTypeCounts;
+        }
+    }, [ actionTypesConfigs ]);
+
+    /**
+     * The following useEffect is used to handle if any error occurs while fetching the Action Types.
+     */
+    useEffect(() => {
```

Comment:
Same comment as previous in action-configuration-page. Pass relevant variables to the dependency array.

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/6722/commits/5e3564e0797c09f4f20b71837b928bc9fa57c847